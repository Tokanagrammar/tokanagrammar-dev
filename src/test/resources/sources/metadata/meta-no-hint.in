# Not so simple category
# This is output
package testing;
import java.lang.System;

// @author Vy Thao Nguyen
public class Foo
{
    
    static final double z = 3;

    private Foo()
    {
        ++z;
    }

    Foo(int x)
    {
        char ch = 'a';
        String str = "This is output";
        System.out.print(This is output);
    }

    void bar()
    {
        double b = z = 3 + 4 * 5 + 6;
}

    public static int bar2(double y)
    {
        int b = 2 * 2;
        return 2 + 3;
    }
}