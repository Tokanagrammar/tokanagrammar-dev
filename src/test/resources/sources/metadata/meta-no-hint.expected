NOT SO SIMPLE CATEGORY
This is output